hw1/02-palindrome.js

```diff
@@ -1,3 +1,28 @@
-const elem = document.querySelector('input');
+document.addEventListener('DOMContentLoaded', () => {
+  function isPalindrome(numberInt) {
+    const numberStr = numberInt.toString();
+    const numberStrReversed = numberStr.split('').reverse().join('');
+    return numberStr === numberStrReversed;
+  }
 
-elem.addEventListener('input', handleInput);
+  const inputElement = document.getElementById('inputID');
+  const outputElement = document.getElementById('outputID');
+
+  inputElement.addEventListener('keydown', (event) => {
+    if (event.key === 'Enter') {
+      const numberStr = inputElement.value;
+      const numberInt = parseInt(numberStr, 10);
+
+      if (Number.isSafeInteger(numberInt) && numberInt >= 0) {
+        const isNumberPalindrome = isPalindrome(numberInt);
+        outputElement.innerHTML = isNumberPalindrome
+          ? '<p class="m-0" style="color: Green;">Yes, This is a Palindrome!</p>'
```

Comment:
since she wants us to review the classes in bootstrap, I found out: 
Red
.text-success
Green
.text-danger
Yellow
.text-warning